analysis_options.yaml

```diff
@@ -167,7 +167,7 @@ linter:
     - slash_for_doc_comments
     # - sort_child_properties_last # not yet tested
     - sort_constructors_first
-    - sort_pub_dependencies
+    # - sort_pub_dependencies
```

Comment:
```suggestion
```

This was added in line 215, so remove commented-out line?

Reply:
This is what's in the Flutter version that I hope to eventually actually share, so I wanted it to match.